catroid/src/main/java/org/catrobat/catroid/stage/ShowTextActor.java

```diff
@@ -116,6 +116,8 @@ private void drawVariables(List<UserVariable> variableList, Batch batch) {
 					if (variable.getVisible()) {
 						if (isNumberAndInteger(variableValue)) {
 							drawText(batch, variableValueWithoutDecimal, xPosition, yPosition);
+						} else if (variable.getValue().toString().isEmpty()) {
+							drawText(batch, Constants.NO_VALUE_SET, xPosition - 200, yPosition);
```

Comment:
I'm not sure about the `-200`.
If you list multiple variables (one below the other), they won't be arranged nicely. Also the positioning will be inconsistent with all the other options (number, text, no variable set)